server/channels/manualtesting/manual_testing.go

```diff
@@ -142,6 +153,11 @@ func ManualTest(c *web.Context, w http.ResponseWriter, r *http.Request) {
 			HttpOnly: true,
 		}
 		http.SetCookie(w, sessionCookie)
+		if err := r.ParseForm(); err != nil {
+			c.Logger.Error("Error parsing form during redirect", mlog.Err(err))
+			c.Err = model.NewAppError("manualTest", "manaultesting.manual_test.parse_form.app_error", nil, "", http.StatusBadRequest).Wrap(err)
```

Comment:
Are these new errors? If so, you need to run `make i18n-extract` to add them to the translation file.